src/Turing.jl

```diff
@@ -24,30 +24,29 @@ using Markdown
 using Libtask
 using MacroTools
 
-function __init__()
-    @require CmdStan="593b3428-ca2f-500c-ae53-031589ec8ddd" @eval begin
-        using CmdStan
-        import CmdStan: Adapt, Hmc
-    end
+@init @require CmdStan="593b3428-ca2f-500c-ae53-031589ec8ddd" @eval begin
+    using CmdStan
+    import CmdStan: Adapt, Hmc
+end
 
-    @require DynamicHMC="bbc10e6e-7c05-544b-b16e-64fede858acb" @eval begin
-        using DynamicHMC, LogDensityProblems
-        using LogDensityProblems: AbstractLogDensityProblem, ValueGradient
+@init @require DynamicHMC="bbc10e6e-7c05-544b-b16e-64fede858acb" @eval begin
+    using .DynamicHMC: NUTS_init_tune_mcmc
+    using LogDensityProblems: LogDensityProblems, AbstractLogDensityProblem, ValueGradient
 
-        struct FunctionLogDensity{F} <: AbstractLogDensityProblem
-          dimension::Int
-          f::F
-        end
+    struct FunctionLogDensity{F} <: AbstractLogDensityProblem
+      dimension::Int
+      f::F
+    end
 
-        LogDensityProblems.dimension(ℓ::FunctionLogDensity) = ℓ.dimension
+    LogDensityProblems.dimension(ℓ::FunctionLogDensity) = ℓ.dimension
 
-        LogDensityProblems.logdensity(::Type{ValueGradient}, ℓ::FunctionLogDensity, x) = ℓ.f(x)::ValueGradient
-    end
+    LogDensityProblems.logdensity(::Type{ValueGradient}, ℓ::FunctionLogDensity, x) = ℓ.f(x)::ValueGradient
 end
+
 import Base: ~, convert, promote_rule, rand, getindex, setindex!
 import Distributions: sample
-import ForwardDiff: gradient
-using Flux.Tracker
+using ForwardDiff: ForwardDiff
```

Comment:
Maybe silly, what's the diff between `using ForwardDiff: ForwardDiff` and `using ForwardDiff`?

Reply:
`using ForwardDiff` brings in all `names(ForwardDiff)` to the namespace which are the all exports. `using ForwardDiff: ForwardDiff` is similar to `import ForwardDiff` so it only brings in the module name `ForwardDiff` to the namespace. I will change it to `import` if that's more readable.

Reply:
Thanks, `using ForwardDiff: ForwardDiff` works fine.